src/core/PerspectivesController.ts

```diff
@@ -24,28 +24,14 @@ export default class PerspectivesController {
         const FILENAME = 'perspectives.json'
         const FILEPATH = path.join(rootConfigPath, FILENAME)
 
-        const neighbourhoods = 'neighbourhoods.json'
-        const filepathNeighbourhoods = path.join(rootConfigPath, neighbourhoods);
-
         if(fs.existsSync(FILEPATH)) {
             const fileObject = JSON.parse(fs.readFileSync(FILEPATH).toString())
-            let neighbourhoodsFileObject = null as Object | null;
-            if(fs.existsSync(filepathNeighbourhoods))  {
-                neighbourhoodsFileObject = JSON.parse(fs.readFileSync(filepathNeighbourhoods).toString())
-            }
 
             const entries = Object.keys(fileObject).map(k => {
-                console.debug(`Found Perspective "${k}":`, fileObject[k])
-                this.#perspectiveHandles.set(k, fileObject[k])
-                if (neighbourhoodsFileObject) {
-                    //@ts-ignore
-                    let neighbourhood = neighbourhoodsFileObject[k];
-                    if (neighbourhood != {}) {
-                        let neighbourhoodTyped = neighbourhood as Neighbourhood;
-                        console.debug(`Found neighbourhood for perspective "${k}":`, neighbourhoodTyped)
-                        this.#perspectiveInstances.set(k, new Perspective(fileObject[k], this.#context, neighbourhoodTyped))
-                    }
-                }
```

Comment:
Is this correct? You're removing the code that was recently added..

Reply:
Yep having neighbourhood on the PerspectiveHandle means we dont need another .json file to store neighbourhood data as it is done on the PerspectiveHandle objects already

Reply:
ah, perfect